### PR TITLE
scripts/file_operations.py - function create_directory() added

### PR DESCRIPTION
**Pull request by braun-devwtf**
### Background

I wanted to use Auto-GPT for programming tasks. For testing I wanted to let it code a website in a project folder. 
But already with the creation of the project folder I ran again and again in error message. 
Auto-GPT tried to find workarounds and create the folder via executing python. 
Then I looked at your source code and found that I just need to extend the commands list. While researching I came across the following issue: https://github.com/Torantulino/Auto-GPT/discussions/363
 where I found the code snippet with the make_directory function.

### Changes

scripts/commands.py
modified execute_command() function
See: files changed

scripts/file_operations
added create_directory() function
See: files changed

### Documentation

issue: https://github.com/Torantulino/Auto-GPT/discussions/363 
I implemented this code snippet in the file_operations.py.

Then I had to extend the long if-else construct in commands.py in the execute_command() function so that the make_directory function can be executed. 

to avoid confusion in the naming I renamed the function in both places in the code to 'create_directory', adjusted the parameter list and tested the creation of folders with auto-gpt afterwards. 

### Test Plan

The testing was done by starting Auto-GPT and telling it to create a folder with the create_directory command.
See 

[log_terminal.md](https://github.com/Torantulino/Auto-GPT/files/11232656/log_terminal.md)
screenshoot1_Terminal_auto-gpt_create_directory
screenshot2_testFolderCreatedSuccessful

### PR Quality Checklist
- [x] My pull request is atomic and focuses on a single change.
- [x] I have thoroughly tested my changes with multiple different prompts.
- [x] I have considered potential risks and mitigations for my changes.
- [x] I have documented my changes clearly and comprehensively.
- [x] I have not snuck in any "extra" small tweaks changes

### CODE REVIEW

**Comment by braun-devwtf** [comment]
It is late in Germany will write documentation later if it is need to approve pull request

File: ai_settings.yaml
Comment (nponeccop) [inline ai_settings.yaml:1]: This change is unrelated. Revert.

File: notes.md
Comment (nponeccop) [inline notes.md:1]: A new file, why?

File: scripts/commands.py
Comment (nponeccop) [inline scripts/commands.py]: Doesn't look right

**Review by nponeccop — CHANGES_REQUESTED** [review]
Remove unrelated changes

File: notes.md
Comment (braun-viathan) [reply notes.md:1]: A notes file for myself should not be committed to Master sorry.

File: scripts/commands.py
Comment (braun-viathan) [reply scripts/commands.py]: Yeah this where some comments from my development process.

**Comment by braun-devwtf** [comment]
Do I have to remove the unrelated changes or can you do this?

**Comment by nponeccop** [comment]
For now PR authors fix their PRs, as the merging team is busy merging non-problematic PRs. You can see the backlog is pretty long.

**Comment by braun-devwtf** [comment]
I Will lookup, how to remove the unwanted changes from the PR. 
Thank you for answering my question

**Review by braun-devwtf — COMMENTED** [review]
I Think I have removed all unwanted changes, 
If I still got anything incorrect notify me and I will fix the issue. 

Thanks for your patience.

**Review by braun-devwtf — COMMENTED** [review]
Now it should be really only the code I want to change inside this PR.

**Review by nponeccop — APPROVED** [review]

**Comment by nponeccop** [comment]
Ok JFYI the PR is in Batch 5 now and will be merged today or tomorrow (today is the Batch 4 but we merge Batch 5 if we have time) unless rejected by the maintainers by applying a label. There will be likely more conflicts before we finally merge you. Be patient.

**Comment by nponeccop** [comment]
@braun-devwtf There are conflicts now

**Comment by braun-devwtf** [comment]
I have remove the whitespace which cause the error for scripts/file_operations.py in line 46. 
What is the other merging conflict ?

**Review by braun-devwtf — COMMENTED** [review]
whitespace removed

**Comment by nponeccop** [comment]
@braun-devwtf There are still conflicts. Do you need help?

**Comment by braun-devwtf** [comment]
Yes then I need help. 
How can I resolve the conflict ?

**Review by braun-devwtf — COMMENTED** [review]
updated comment but, I don't know how to show the merging conflicts.

**Comment by braun-devwtf** [comment]
Bildschirmfoto 2023-04-15 um 20 12 15
Can not click on resolve conflict here. 
 
Thanks for your patience with me :)

**Comment by nponeccop** [comment]
@braun-devwtf There are conflicts now

**Review by braun-devwtf — COMMENTED** [review]
forgot to import the returned function.
Am I doing this wrong ? asked chatGPT cause I was not able to resolve the conflict via the web interface. 
So I tried to resolve the conflict locally.

**Review by braun-devwtf — COMMENTED** [review]
viewed

**Comment by k-boikov** [comment]
The structure of the PR is totally outdated, these files are not longer in the /scripts folder. I suggest you start from scratch or rebase.

**Pull request closed without merging**
